docs/stylesheets/extra.css

```diff
@@ -0,0 +1,3 @@
+.md-grid {
+    max-width: 75rem
+}
```

Comment:
Another is for text in Tables - Font size is reduced and harder to read. here's a fix (try that first, just in case):

```css
.md-typeset table:not([class]) {
    font-size: 0.75rem
}
```

This should make it more accessible for those with low eyesight - Feel free to experiment with other sizes too

Reply:
This will also enhance the visual aid for accessibility purposes (screen readers), and those with low eyesight or colorblind by adding a visual cue it's a link with a tiny bottom border

```css
.md-typeset a {
    border-bottom: 0.1px dashed black
}
```

Reply:
Quick fix to increase emphasis on active navigation anchors as you scroll the page, and the menu that's being actively viewed - Also for low eyesight

```css
.md-nav__link--active {
    font-weight: bold
}
```

It's subtle, you'll only notice as you scroll the page on your right (right sidemenu)

Reply:
Another one to fix low eyesight issues when trying to read non-bold text in warnings, info, boxes, etc.

```css
.md-typeset .admonition, .md-typeset details {
    font-size: 0.70rem
}
```

Maybe needs !important, not sure, worth checking

Reply:
added proposed changes